index.html

```diff
@@ -1718,8 +1717,7 @@ <h3>Names and Descriptions</h3>
       "@language": "fr"
     }, {
       "@value": "جامعة عامة تركز على أمثلة التدريس.",
-      "@language": "ar",
-      "@direction": "rtl"
+      "@language": "ar"
```

Comment:
I left a comment on #1424. Notice that this "purely RTL string" does not display correctly in the diff--because it is in an LTR context and ends with a neutral character (`.`). The `.` is on the wrong side of the string.

Reply:
GitHub won't handle a suggestion on a line range that includes deleted/changed lines, so I'm making a change suggestion on 1719 above -- to move the `.` to the correct side of the string.